tests/ft_php.vader

```diff
@@ -44,5 +47,7 @@ Execute (phpmd):
   AssertEqual len(loclist), 2
   AssertEqual loclist[0].lnum, 16
   AssertEqual loclist[0].text, "Avoid unused parameters such as '$request'."
+  AssertEqual loclist[0].type, 'E'
   AssertEqual loclist[1].lnum, 48
   AssertEqual loclist[1].text, "Avoid variables with short names like $id. Configured minimumlength is 3."
+  AssertEqual loclist[1].type, 'E'
```

Comment:
Is this something we need to change in this PR or should I create another? Otherwise changes LGTM